can_tools/bootstrap_data/covid_demographics.csv

```diff
@@ -35,3 +35,5 @@ all,all,unknown,all
 all,all,all,male
 all,all,all,female
 all,all,all,other
+all,latinx,all,all
```

Comment:
Looking at the actual CTP data, it seems like `latinx` lines up exactly with `ethnicity=hispanic`.

Is there an added bonus to having this?

Reply:
I just put it here to report the data as is. 

I think this is an "official" (from census or some other federal resource) classification for race and/or ethnicity. Mirroring the official is what I went after